RELEASE/scripts/autoscend/quests/level_12.ash

```diff
@@ -1913,6 +1913,7 @@ boolean L12_themtharHills()
 	buffMaintain($effect[Sweet Heart], 0, 1, 20);
 	buffMaintain($effect[Good Things Are Coming, You Can Smell It]);
 	buffMaintain($effect[Incredibly Well Lit]);
+	auto_getCitizenZone("meat");
```

Comment:
since this buff can take a turn, move it to be first buff we get. Don't want to use up a turn of the other ones

Reply:
It can unless we use a free runaway. Good idea though in case we don't have a free run available